deepcell/model_zoo/__init__.py

```diff
@@ -34,7 +34,8 @@
 from deepcell.model_zoo.featurenet import bn_feature_net_3D
 from deepcell.model_zoo.featurenet import bn_feature_net_skip_3D
 
-from deepcell.model_zoo.featurenet import siamese_model
+from deepcell.model_zoo.tracking import siamese_model
```

Comment:
Since you're changing the import path, make sure this is not disrupted in the existing tracking notebook.

Reply:
Both tracking notebooks (training and benchmarking will need to be updated for the new approach. Should be added before releasing the new preprint, but can be held until then.